<launch>

  <arg name="marker_size" default="0.01" />
  <arg name="marker_dictionary" default="TAG36h11" />

  <remap from="/image" to="/cameras/left_hand_camera/image"/>
  <remap from="/camera_info" to="/cam2/camera_info"/>

  <include file="$(find success_ros_aruco)/launch/base_aruco_node.launch">
    
    <arg name="marker_dictionary" value="$(arg marker_dictionary)"/>
    <arg name="marker_size" value="$(arg marker_size)"/>
    <arg name="rate" value="10"/>

  </include>

</launch>
